resteasy-core/src/main/java/org/jboss/resteasy/ResteasyThreadContext.java

```diff
@@ -17,7 +17,7 @@
  * limitations under the License.
  */
 
-package org.jboss.resteasy.concurrent;
+package org.jboss.resteasy;
```

Comment:
I think we should move it to `org.jboss.resteasy.core.concurrent` to ensure we don't split packages elsewhere.